sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -20,7 +20,7 @@ public class MetricsAdvisorAdministrationClient
     {
         private readonly ClientDiagnostics _clientDiagnostics;
 
-        private readonly AzureCognitiveServiceMetricsAdvisorRestAPIOpenAPIV2RestClient _serviceRestClient;
+        private readonly MicrosoftAzureMetricsAdvisorRestAPIOpenAPIV2RestClient _serviceRestClient;
```

Comment:
is there a way to tell codegen to create a service rest client with a better name? this is kind of :|

Reply:
I didn't look through the new model factory generator, so I'm not sure. I didn't want to worry about the factory now since it's not high priority for the next preview. I'll see if I can hide it somehow for now.

Reply:
You can always create an issue for this and make sure that gets a proper rename.
My understanding is that you can rename it with a custom class like with any generated class.

Also, so there is a preview and then another release for the GA? so this code complete doesn't have to be GA ready?

Reply:
Correct, but we wanted to get the final design closed in this preview if possible. We're still waiting feedback from architects, though, so some changes are expected after this release.

Reply:
Hid the factory for now. Being tracked with https://github.com/Azure/azure-sdk-for-net/issues/20554.